viewer/packages/api/visual-tests/TwoModels.VisualTest.ts

```diff
@@ -1,22 +1,69 @@
 /*!
  * Copyright 2022 Cognite AS
  */
-import { CogniteCadModel } from '..';
+import { CogniteCadModel, Cognite3DViewer, CogniteModel } from '..';
 import * as THREE from 'three';
 
-import {
-  ViewerTestFixtureComponents,
-  ViewerVisualTestFixture
-} from '../../../visual-tests/test-fixtures/ViewerVisualTestFixture';
+import { VisualTestFixture } from '../../../visual-tests/test-fixtures/VisualTestFixture';
 import { DefaultNodeAppearance } from '@reveal/cad-styling';
+import { createCognite3DViewer } from '../../../visual-tests/test-fixtures/utilities/cognite3DViewerHelpers';
+import { AxisViewTool } from '../../../packages/tools';
+
+export default class TwoModelsVisualTest implements VisualTestFixture {
+  private _viewer!: Cognite3DViewer;
+  private readonly _renderer: THREE.WebGLRenderer;
 
-export default class TwoModelsVisualTest extends ViewerVisualTestFixture {
   constructor() {
-    super('primitives', 'primitives');
+    this._renderer = new THREE.WebGLRenderer({ powerPreference: 'high-performance' });
+    this._renderer.setPixelRatio(window.devicePixelRatio);
   }
-  public setup(testFixtureComponents: ViewerTestFixtureComponents): Promise<void> {
-    const { models } = testFixtureComponents;
 
+  public async run(): Promise<void> {
+    let totalItemsLoaded = 0;
+    let totalItemsRequested = 0;
+
+    this._viewer = await createCognite3DViewer(modelLoadingCallback, this._renderer);
+
+    this.setupDom(this._viewer);
+
+    // Load models SEQUENTIALLY to avoid resource conflicts
+    const firstModel = await this._viewer.addCadModel({
```

Comment:
Why do we need to make sure they're loaded serially? Can't we just load both at the same time?